web-module/src/main/webapp/resources/celJS/images/imageSlideShow.js

```diff
@@ -941,10 +945,128 @@ window.CELEMENTS.image.SlideShow = function(config) {
       _startStopClickHandler : function(event) {
         var _me = this;
         event.stop();
-        _me.startStop();
+        var clickedElement = event.findElement();
+        var linkHref = '';
+        if(clickedElement.up('div.cel_slideShow_slideWrapper a')) {
+          linkHref = clickedElement.up('div.cel_slideShow_slideWrapper a').href;
+        }
+        $(document.body).observe('click', _me._contextMenuSlideShowListItemClickedBind);
```

Comment:
There will be one more "observer" registration every start-, stop-click. Please add a stopObserving before.